packages/env/README.md

```diff
@@ -0,0 +1,65 @@
+# `@wordpress/env`
+
+> A zero-config, self contained local WordPress environment for development and testing.
+
+## Usage
+
+```sh
+$ npm -g i @wordpress/env
+
+$ cd path/to/gutenberg # WordPress install will be in path/to/gutenberg-wordpress.
```

Comment:
Shouldn't it be
> $ cd path/to/plugin-or-theme # WordPress install will be in path/to/plugin-or-theme-wordpress.

Reply:
Yeah, I'll add that with the PR that adds theme support.